core/src/main/java/org/apache/iceberg/RemoveSnapshots.java

```diff
@@ -321,8 +321,6 @@ public void commit() {
 
     if (cleanExpiredFiles) {
       cleanExpiredSnapshots();
```

Comment:
Inside `cleanExpiredSnapshots()` logs are present, which are enough to know whether files are cleaned or not.

Reply:
I agree with @rdblue that we should just add the message right above here, "Local delete of expired files enabled. Cleaning up expired manifests and data files" or something like that

Reply:
ok. Added.